### DISCUSSION TITLE
ngrok offline?

### DISCUSSION THREAD

**Discussion by meunumerotim333** [TheLastBen/fast-stable-diffusion#1965] [Q&A]
Getting this error trying to connect on ngrok servers:

![image](https://user-images.githubusercontent.com/121369033/230794921-5934144c-bf90-468b-a60d-e6cc56416dd3.png)

**Comment by meunumerotim333** [comment]
For now I'm back to gradio experience.. and I forget how painful this is 

![image](https://user-images.githubusercontent.com/121369033/230834509-da22d3f5-c2b1-489f-b730-8aa275328869.png)


![image](https://user-images.githubusercontent.com/121369033/230831765-0e23bc02-e488-4d7a-93f8-81575815a9a1.png)

**Comment by meunumerotim333** [comment]
It's back online.

**Comment by TheLastBen** [comment]
great !
